l1-contracts/contracts/common/InitializableRandomStorage.sol

```diff
@@ -0,0 +1,181 @@
+// SPDX-License-Identifier: MIT
+// OpenZeppelin Contracts (last updated v4.6.0) (proxy/utils/Initializable.sol)
```

Comment:
Shall we use the same OZ contracts version as we use in the repo?
- https://github.com/OpenZeppelin/openzeppelin-contracts/blob/v4.8.0/contracts/proxy/utils/Initializable.sol#L2

Reply:
Done